prod_inventory_app/templates/prod_inventory_app/all_sales.html

```diff
@@ -13,13 +13,13 @@ <h1 class="text-center">SALES</h1><br>
         </tr>
         </thead>
     </table>
-{% if messages %}
+    {% if messages %}
     {% for message in messages %}
-        <div>
-            <strong>{{message|safe}}</strong>
-        </div>
+    <div>
+        <strong>{{message|safe}}</strong>
+    </div>
     {% endfor %}
```

Comment:
Line 17 to 20 should be tabbed over since it is inside the if condition. Line 18 to 20 should be tabbed over once more since it is inside the for loop.